docs/articles/vs-test-adapter/Known-Problems.md

```diff
@@ -1,24 +1,30 @@
 # Known Problems
 
-1. There is no test status in Visual Studio corresponding to NUnit's Inconclusive result, so tests with this result are reported as Not Run. Click on the individual test to see the result.
+1. Using both the VSIX adapter and nuget adapters at the same time can lead to discovery issues, ref [Issue769](https://github.com/nunit/nunit3-vs-adapter/issues/769).  The resolution is to uninstall the VSIX adapter, and only use the nuget adapters.  The VSIX adapters are being deprecated in later VS versions.
 
-2. Theories are reported as individual cases, rather as a single unit.
+2. Support for Explicit keyword
 
-3. In NUnit, tests have names, which are not necessarily unique. Visual Studio wants the names to be unique. So if two tests have the same name, VS displays a warning message in the output window. The message may be ignored. Two separate results will be shown under the single test in the explorer pane.
+    1 Adapter versions in the 3.XX series:  From version 16.2 (Visual Studio 2019) the Explicit keyword is no longer explicitly supported by Visual Studio.  If you want to use it, it does exist as a Category also, so you can use it as a category filter.
```

Comment:
```suggestion
* **Adapter versions in the 3.XX series**:  From version 16.2 (Visual Studio 2019) the `explicit` keyword is no longer explicitly supported by Visual Studio.  If you want to use it, it does exist as a Category also, so you can use it as a category filter.
```

Reply:
Part of the suggested change is incorrect: the NUnit keyword is spelled `Explicit` not `explicit`. Have to beware of what may seem like formatting changes (casing) but are actually content changes.

Reply:
Thanks @CharliePoole -- this suggestion was made for consistency with another change: https://github.com/nunit/docs/pull/472#discussion_r458043193

It looks like I chose the wrong version to be consistent with. 😄